app/repository/pg/user.go

```diff
@@ -49,3 +50,20 @@ func (r Repo) Add(user entities.User) (string, error) {
 
 	return id, nil
 }
+
+// GetUserByID meth implements usecase.UserRepository logic
+// finding user in DB by ID.
+func (r Repo) Find(id string) (entities.PublicUser, error) {
+	var user entities.PublicUser
+
+	const sqlStatement = `SELECT id, first_name, last_name, email, created_at FROM "user" WHERE id=$1`
```

Comment:
But what if the user with the id specified in the request has already been soft deleted? Will it still be returned?

Reply:
"Deleted user should not be returned" it's written in the task

Reply:
We can't use not existed  fields and entities. We use it after they will appear

Reply:
> "Deleted user should not be returned" it's written in the task
Andriy said at a lecture about this, yesterday )

Reply:
So far, it has not been possible to view the lecture. But if you claim that this is how it should be, then I will take your word for it =)